commodore/component-template/{{ cookiecutter.slug }}/Makefile

```diff
@@ -42,10 +42,11 @@ docs-serve: ## Preview the documentation
 
 .PHONY: compile
 .compile:
+	mkdir -p dependencies
```

Comment:
Doesn't seem required according to my local tests (Linux), can you elaborate why you think this is needed?

Reply:
On "my machine"™️ (macos) commodore complains
```sh
$ docker run --rm -u "$(id -u)" -w /keycloak -v "${PWD}:/keycloak" projectsyn/commodore:latest component compile . -f tests/builtin.yml --search-paths ./dependencies
Error: Invalid value for '-J' / '--search-paths': Directory './dependencies' does not exist.
$ docker run --rm -u "$(id -u)" -w /keycloak -v "${PWD}:/keycloak" projectsyn/commodore:latest --version
commodore, version 0.8.2 (Git version: v0.8.2-6-g1788a4a)
```

Reply:
Ah yes, I was not able to properly test it, I get the same issue when the directory really doesn't exist :see_no_evil: